components/form/FAutocompleteInput.vue

```diff
@@ -364,6 +373,15 @@ function forceValidation() {
 
 watch(isValid, forceValidation);
 
+const FMenuRef = ref()
+
+onMounted(() => {
+  const matchingOption = props.options.find(
+    option => stringify(props.modelValue) === stringify(option.value)
+  );
+  FMenuRef.value.selectOption(matchingOption ?? null)
```

Comment:
you need to make sure the ref is clearly mounted (I wonder how typescript did not complain):
```suggestion
  FMenuRef.value?.selectOption(matchingOption ?? null)
```

Reply:
No, in fact we are never 100% sure that all the children are mounted, even if the parent is